lib/order.rb

```diff
@@ -0,0 +1,33 @@
+require 'menu'
```

Comment:
I couldn't feature test in irb because this file wouldn't load. You need to use 'require_relative' to make that possible.